### PR TITLE
Task/13 create notification screen

### PR DESCRIPTION
**Pull request by diksha-mobile-dev**

### CODE REVIEW

File: lib/time_difference.dart [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import 'package:timeago/timeago.dart' as timeago;
+
+class TimeDifference {
+  static String postAddedTimeDifference(int timeStamp) {
+    DateTime now = DateTime.now();
+    var date1 = new DateTime.fromMillisecondsSinceEpoch(timeStamp * 1000);
+    final diff = now.difference(date1).inMilliseconds;
+    final fifteenAgo =
```
Comment (santhosh-mobile-dev) [inline lib/time_difference.dart]: What is mean by firteenAgo ?? Don’t copy paste code as it is .. change variable name according to meaning

File: lib/ui/screen/create_account.dart [hunk 2]
```diff
@@ -1,13 +1,13 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:genericwidgetapp/custom_message.dart';
+import 'package:genericwidgetapp/ui/screen/login.dart';
 import 'package:genericwidgetapp/ui/widgets/custom_label_text.dart';
 
+import '../widgets/custom_flat_button.dart';
```
Comment (santhosh-mobile-dev) [inline lib/ui/screen/create_account.dart]: What’s this.. don’t import path like this .. import absolute path ..

File: lib/ui/screen/notifications_screen.dart [hunk 3]
```diff
@@ -1,56 +1,218 @@
...
+                            .name} have rejected the boost' )
+                            : Container( ),
+                      ],
+                    ),
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
```
Comment (santhosh-mobile-dev) [inline lib/ui/screen/notifications_screen.dart:161]: Why build item have 100 lines of code .. why it’s not refactored widget by widget ?

File: lib/ui/screen/notifications_screen.dart [hunk 4]
```diff
@@ -1,56 +1,218 @@
...
-  @override
-  Widget build(BuildContext context) {
-    print(dataSource.notifications.length);
+  Widget body(BuildContext context) {
     return Scaffold(
       body: dataSource.notifications.length == 0
-          ? Center(child: CircularProgressIndicator())
-          :  ListView.separated(
-        padding: EdgeInsets.symmetric(vertical: 8.0),
+          ? Center( child: CircularProgressIndicator( ) )
```
Comment (santhosh-mobile-dev) [inline lib/ui/screen/notifications_screen.dart]: Why it’s not created seperately

File: lib/ui/widgets/custom_list_tile.dart [hunk 5]
```diff
@@ -4,13 +4,20 @@ class CustomListTile extends StatelessWidget {
   final IconData iconData;
   final String text;
 
-  CustomListTile({this.iconData,this.text});
+  CustomListTile({this.iconData, this.text});
```
Comment (santhosh-mobile-dev) [inline lib/ui/widgets/custom_list_tile.dart:7]: What is this for ?

File: lib/ui/widgets/notification_alert.dart [hunk 6]
```diff
@@ -0,0 +1,25 @@
+import 'package:flutter/material.dart';
+
+class NotiDelete extends StatelessWidget {
```
Comment (santhosh-mobile-dev) [inline lib/ui/widgets/notification_alert.dart]: What’s the name ?? This is a dialog .. it should be named properlu .. NotificationDeleteDialog

**Review by santhosh-mobile-dev — CHANGES_REQUESTED** [review]
See inline comments and make changes .. @diksha-mobile-dev learn naming conversions properly .. don’t write 100 lines of nested widgets ..

File: lib/ui/widgets/custom_list_tile.dart (hunk 5 above)
Comment (diksha-mobile-dev) [reply lib/ui/widgets/custom_list_tile.dart:7]: this is been used in SideMenu which is not created by me

File: lib/ui/widgets/notification_alert.dart (hunk 6 above)
Comment (diksha-mobile-dev) [reply lib/ui/widgets/notification_alert.dart]: This page is not used anywhere, I forget to remove it.

**Pull request merged**